ibis/backends/druid/__init__.py

```diff
@@ -130,11 +133,17 @@ def get_schema(
         catalog: str | None = None,
         database: str | None = None,
     ) -> sch.Schema:
-        return self._get_schema_using_query(
-            sg.select(STAR)
-            .from_(sg.table(table_name, db=database, catalog=catalog))
-            .sql(self.dialect)
-        )
+        try:
+            schema = self._get_schema_using_query(
+                sg.select(STAR)
+                .from_(sg.table(table_name, db=database, catalog=catalog))
+                .sql(self.dialect)
+            )
+        except ProgrammingError as e:
+            if re.search(r"\bINVALID_INPUT\b", str(e)):
```

Comment:
I wasn't able to get anything better than this matching, if anyone has any ideas, suggestions welcome